# Request
string entity_name
---
# Response
bool success
string message